frontend/dashboard/pages/OrgContentLibrary/OrgContentLibrary.tsx

```diff
@@ -25,6 +32,29 @@ export function OrgContentLibrary(): ReactElement {
 }
 
 function OrgContentLibraryWithContext(): ReactElement {
+  const { t } = useTranslation();
+  const selectedContext = useSelectedContext();
+
+  const { data: codeListsResponseList, status: codeListDataListStatus } =
+    useOrgCodeListsQuery(selectedContext);
+
+  switch (codeListDataListStatus) {
+    case 'pending':
+      return <StudioPageSpinner spinnerTitle={t('general.loading')} />;
+    case 'error':
+      return <StudioPageError message={t('dashboard.org_library.fetch_error')} />;
+    case 'success':
+      return <OrgContentLibraryWithContextAndData codeListsData={codeListsResponseList} />;
+  }
+}
+
+type OrgContentLibraryWithContextAndDataProps = {
+  codeListsData: CodeListData[];
+};
```

Comment:
Naming the variable `codeListData` might make people assume that it has the type `CodeListData` and not `CodeListData[]`. I suggest naming it `CodeListDataList` to avoid possible future confusion.

```suggestion
type OrgContentLibraryWithContextAndDataProps = {
  codeListsDataList: CodeListData[];
};
```